codalab/rest/workers.py

```diff
@@ -34,7 +33,7 @@ def checkin(worker_id):
         request.json.get("memory_bytes"),
         request.json.get("free_disk_bytes"),
         request.json["dependencies"],
-        request.json.get("shared_file_system"),
+        request.json.get("shared_file_system", False),
```

Comment:
For consistency with the other fields, could we just pass `None` back?

Reply:
The database schema for this field does not accept null values. The cleanest solution would be to update all the workers so they simply report this field, but if that's not possible defaulting to False is the right thing to do.